doc/nrf/ug_partition_manager.rst

```diff
@@ -279,3 +279,72 @@ CMake usage
 
       --slot-size $<TARGET_PROPERTY:partition_manager,MCUBOOT_SLOT_SIZE>
 
+.. _ug_pm_predefined:
+
+Predefined configuration
+========================
+Predefined configuration is a feature intended for deployed products which consists of multiple images where
+only a subset of the included images will be upgradeable through firmware update mechanisms.
+One example of this is a device which includes a non-upgradable first stage bootloader and an upgradeable application.
+In these use cases, the images to be upgraded must be linked to the same address as that which is deployed.
+This to avoid accidental overrides, and incorrect configurations.
+
+Predefined configuration allows the user to configure the **dynamic partition**.
```

Comment:
Since we are using the word dynamic it seems unnatural to not use the word static as it's counterpart.

Reply:
Hm, using the word "partition" here might be confusing. It is only supposed to describe the area within which changes may take place (resizing etc). Maybe "dynamic area" is better? 

I struggle to find a good term to describe this so any proposals are welcome.

Reply:
Updated to "dynamic area" in which case I don't see the need to explicitly describe the static counterpart.